src/main/java/hobbylist/logic/parser/AddCommandParser.java

```diff
@@ -25,9 +26,11 @@ public class AddCommandParser implements Parser<AddCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public AddCommand parse(String args) throws ParseException {
+        boolean isStatusPresent = args.matches(".*\\bs/.*");
```

Comment:
I think there is no need for this line. argMultimap.getValue returns an optional which you can use to check whether it is empty

Reply:
I agree

Reply:
I have removed this line and use isPresent() to check if CliSyntax is there or not instead.